ci-operator/step-registry/gather/must-gather/gather-must-gather-commands.sh

```
@@ -313,9 +313,16 @@ fi
 
 MUST_GATHER_TIMEOUT=${MUST_GATHER_TIMEOUT:-"15m"}
 
+# no internet access in C2S/SC2S env, disable proxy
+if [[ "${CLUSTER_TYPE:-}" =~ ^aws-s?c2s$ ]]; then
+    source "${SHARED_DIR}/unset-proxy.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./unset-proxy.sh was not specified as input (see shellcheck -x).

(SC1091)